portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase

```diff
@@ -966,6 +966,93 @@ definition {
 		}
 	}
 
+	test MultipleElasticsearchConnections {
+		property osgi.app.includes = "portal-search-elasticsearch7";
+		property osgi.module.configuration.file.names = "com.liferay.portal.bundle.blacklist.internal.BundleBlacklistConfiguration.config";
+		property osgi.module.configurations = "blacklistBundleSymbolicNames=&quot;com.liferay.portal.mobile.device.detection.fiftyonedegrees.api,com.liferay.portal.mobile.device.detection.fiftyonedegrees.enterprise,com.liferay.portal.mobile.device.detection.fiftyonedegrees.enterprise.test.data,com.liferay.portal.mobile.device.detection.fiftyonedegrees,com.liferay.portal.search.elasticsearch6.impl,com.liferay.portal.search.elasticsearch6.spi,com.liferay.portal.search.elasticsearch6.api,com.liferay.portal.search.elasticsearch6.xpack.security.impl,com.liferay.portal.search.elasticsearch6.xpack.monitoring.web,com.liferay.portal.search.tuning.rankings.web&quot;";
+		property portal.upstream = "quarantine";
+		property remote.elasticsearch.enabled = "true";
+		property remote.elasticsearch.clusters.enabled = "true";
```

Comment:
The reindex test for CCR was added after this PR was created. Can you change the property for that test as well and squash it into your rename commit?